website/docs/build-apps/how-to-guides/Server-side-filtering-table.md

```diff
@@ -1,23 +1,64 @@
-
+---
+description: This page shows you how to set up server-side filtering on a Table widget, which allows you to refine query results based on specific criteria.
+---
 
 # Setup Server-side Filtering on Table
```

Comment:
There are two guides for server-side filtering on the table. This is available on docs too.
<img width="305" alt="Screenshot 2023-09-29 at 2 06 32 PM" src="https://github.com/appsmithorg/appsmith-docs/assets/99252011/6892bcc8-dfb5-42c7-9e64-20dfb180bcf7">